ansible/roles/debops.prosody/defaults/main.yml

```diff
@@ -0,0 +1,162 @@
+---
+
+
+prosody__deploy_state: "present"
+
+
+# Packages and installation [[[
+# -----------------------------
+
+# .. envvar:: apache__base_packages [[[
+#
+# List of base packages to install.
+prosody__base_packages:
+  - prosody
+  - lua-zlib
+  - prosody-modules # for ldap auth
+#  - lua-event # for use_libevent = true
+                                                                   # ]]]
+                                                                   # ]]]
+
+
```

Comment:
Whitespace between variables like this should be avoided to make the vim folded view more readable.